app/assets/javascripts/discourse/tests/integration/components/select-kit/mini-tag-chooser-test.js

```diff
@@ -206,6 +206,25 @@ module(
       assert.dom(".select-kit-collection").doesNotExist();
       assert.dom(".select-kit-header button[data-name='apple']").doesNotExist();
       assert.strictEqual(this.subject.header().value(), "orange,potato");
+
+      const filterInput = document.querySelector(
+        ".select-kit-header .filter-input"
+      );
+
+      assert.strictEqual(
+        filterInput.getAttribute("placeholder"),
+        "",
+        "Placeholder is empty when there is a selection"
+      );
```

Comment:
a dash of qunit-dom sprinkles 😉 

```suggestion
      assert
        .dom(".select-kit-header .filter-input")
        .hasAttribute(
          "placeholder",
          "",
          "Placeholder is empty when there is a selection"
        );
```

Reply:
Ah, `dom.().hasAttribute`, nice, thank you!